### ISSUE TITLE
Accessibility of userlist table should be improved

### ISSUE THREAD

**Issue by Volker-E** [owncloud/core#15396]
The current implementation of the `userlist` table (templates/users/part.userlist.php) misses `scope` attributes on the `th` elements. Moreover there should be one `th` per row. IMHO it should be the name element.

Further information at: http://webaim.org/techniques/tables/data

**Comment by Volker-E** [comment]
Connected to #13708

**Comment by jancborchardt** [comment]
Thanks a bunch @Volker-E! :) It’s cool to have you on board regarding accessibility!
